examples/conf.py

```diff
@@ -234,6 +234,7 @@ def setup(app: Sphinx):
     "numpy": ("https://numpy.org/doc/stable/", None),
     "pandas": ("https://pandas.pydata.org/pandas-docs/stable/", None),
     "pymc": ("https://www.pymc.io/projects/docs/en/stable/", None),
+    "pytensor": ("https://pytensor.readthedocs.io/en/latest/", None),
```

Comment:
We should remove aesara from this dict and replace it with pytensor instead of adding pytensor. Same goes for the pre-commit config